independent-publisher-connectors/Airtable/apiDefinition.swagger.json

```diff
@@ -0,0 +1,490 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Airtable",
+    "description": "Airtable custom connector to read and write data",
```

Comment:
@last72 Can you please provide more detailed description as it should describe concisely the main purpose and value offered by your connector.

Documentation - [Description of connector](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#description)